fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -611,6 +612,7 @@ ACTOR Future<Void> sendGrvReplies(Future<GetReadVersionReply> replyFuture,
 		}
 		reply.midShardSize = midShardSize;
 		reply.tagThrottleInfo.clear();
+		grvProxyData->ssVersionVectorCache.getDelta(request.maxVersion, reply.ssVersionVectorDelta);
```

Comment:
Shall I make getDelta() return the version vector? I chose to pass-by-reference to avoid an extra version vector copy. Let me know if the following version is going to be easier to read?

reply.ssVersionVectorDelta = grvProxyData->ssVersionVectorCache.getDelta(request.maxVersion);

Reply:
I like it how it is. faster, and not difficult to read.